### PR TITLE
add sessionStorage cache

### PR DESCRIPTION
**Pull request by fsmanuel**
#2 
@twokul see it as a first draft. I used localStorage for the cache as suggested but in the process i felt like localStorage isn't what we need here. The problem with localStorage is that you have to do the bookkeeping (when browser cache expires, etc.). I'm not familiar with the sessionStorage but i guess it's a better bet. I'll add sessionStorage to ember-local-storage and update the PR later. A plain `Ember.Object` could also be a solution but the data gets lost on a reload. I think sessionStorage is what we want.
What do you think?

### CODE REVIEW

File: addon/components/lazy-image.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
 import Ember           from 'ember';
 import ImageLoadMixin  from '../mixins/image-load';
 import InViewportMixin from '../mixins/in-viewport';
+import cache           from '../models/cache';
```
Comment (fsmanuel) [inline addon/components/lazy-image.js]: @twokul should the cache live in `models` or `lib`?

**Comment by fsmanuel** [comment]
I'll fix the tests and add some for the cache when we are done.

File: addon/components/lazy-image.js (hunk 1 above)
Comment (twokul) [inline addon/components/lazy-image.js]: I'd say `lib`

**Comment by twokul** [comment]
@fsmanuel dude this is rad. I'll review closely tomorrow

**Comment by fsmanuel** [comment]
@twokul updated the PR and added a test.

**Comment by twokul** [comment]
@fsmanuel big open source :heart: to you, dude.

**Pull request merged**